dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/TrackerProgramRuleService.java

```diff
@@ -43,7 +43,7 @@
  */
 public interface TrackerProgramRuleService
 {
-    Map<Enrollment, List<RuleEffect>> calculateEnrollmentRuleEffects( TrackerBundle trackerBundle );
+    Map<String, List<RuleEffect>> calculateEnrollmentRuleEffects( TrackerBundle trackerBundle );
```

Comment:
Can you add a small Javadoc to explain what's the return type? I assume the `String` as key of the Map is an Enrollment UID

Reply:
added